src/porepy/numerics/fv/test_coupler_mpfa.py

```diff
@@ -0,0 +1,133 @@
+import numpy as np
+import time
+import scipy.sparse as sps
+import scipy.spatial as spat
+from scipy.sparse.linalg import spsolve
+import matplotlib.pyplot as plt
+
+from porepy.grids import structured, simplex, point_grid
+from porepy.grids.grid import FaceTag
+from porepy.params import bc, second_order_tensor
+
+
+# from viz.plot_grid import plot_grid, save_img
+from porepy.viz.exporter import export_vtk
+
+from porepy.grids.coarsening import *
+from porepy.grids import grid_bucket, remove_grids
+from porepy.fracs import meshing  # split_grid
+
+from porepy.numerics.mixed_dim import coupler
+import copy
+from porepy.numerics.fv import mpfa, tpfa_coupling, tpfa
+from porepy.numerics.mixed_dim import condensation
+
```

Comment:
Reorganize imports (generally), in line with https://www.python.org/dev/peps/pep-0008/#imports